docs/reference/glossary.asciidoc

```diff
@@ -447,6 +447,17 @@ See the {ref}/indices-rollover-index.html[rollover index API].
 // end::rollover-def[]
 --
 
+ifdef::permanently-unreleased-branch[]
+
+[[glossary-rollup]] rollup ::
+// tag::rollup-def[]
+Aggregates an index's time-series data and stores the results in another index.
```

Comment:
Not a big deal, but this term is one we use a lot in ML docs and so I've looked it up often in the style guide: https://brand.elastic.co/302f66895/p/194a3b-writing-style-guide
```suggestion
Aggregates an index's time series data and stores the results in another index.
```

Reply:
Good catch. Thanks @lcawl!